styles/Hero.module.css

```diff
@@ -0,0 +1,86 @@
+.hero {
+    width: 100%;
+    height: 100vh;
+    background-image: linear-gradient(rgb(148, 199, 231, 0.50),rgb(148, 199, 231, 0.50)), url("../public/heroImage.jpg");
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat; 
+}
+
+.hero__block {
+    width: 50%;
+    background-color: #94C7E7;
+    /* clip-path: polygon(0% 0%, 90% 0%, 100% 50%, 90% 100%, 0% 100%);  */
```

Comment:
@LRode having issues trying to shape the box. ;/ when I use Clip-path, it distorts the positions and makes the hero_block overlap the navigation bar